lib/cf_obs_binary_builder/manifest.rb

```diff
@@ -100,11 +100,10 @@ def dependencies_status(package_statuses, stack_mappings)
   # All other stacks in the manifest stays untouched.
   # E.g. Given a manifest with cflinuxfs3 and cflinuxfs2 and the stack_mapppings below,
   # the final manifest should have cflinuxfs2 and cflinuxfs3 deps untouched and the same deps as cflinuxfs2 for
-  # sle12 and opensuse42 and the same deps as cflinuxfs3 for sle15 (sle15).
+  # sle12 and the same deps as cflinuxfs3 for sle15 (sle15).
```

Comment:
also remove the `and`

Reply:
One "and" was removed, the other does sound reasonable.